### PR TITLE
utils_net: support to check network interfaces of remote machine

### PR DESCRIPTION
**Pull request by balamuruhans**
get_sorted_net_if() and get_net_if() can be used to get network
interfaces of remote machines

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by balamuruhans** [comment]
@sathnaga Thank you

File: virttest/utils_net.py [hunk 1]
```diff
@@ -1265,41 +1265,60 @@ def local_runner_status(cmd, timeout=None, shell=False):
...
     :return: List of network interfaces.
     """
-    if runner is None:
-        runner = local_runner
+    cmd = "ip link"
     if state is None:
         state = ".*"
-    cmd = "ip link"
-    result = runner(cmd)
+    if session:
```
Comment (xutian) [inline virttest/utils_net.py]: it doesn't work, if OS installed in remote machine is windows

Comment (balamuruhans) [inline virttest/utils_net.py]: @xutian, this API was already with command "ip link", even without this support for remote machine, I doubt it works for windows.

Comment (xutian) [inline virttest/utils_net.py]: @balamuruhans  right.  original code only works on host, so it linux OS only.  after this change, session maybe a remote windows guest shell session. so if you can add windows session support is better. And wmic is helpful to implement it.

Comment (balamuruhans) [inline virttest/utils_net.py]: @xutian, okay but I dont have env to work with windows

Comment (xutian) [inline virttest/utils_net.py]: @balamuruhans  it better write another function like get_remote_host_net_ifs(). I have a plan to rewrite get_net_ifs and get_sorted_net_ifs with netifaces module to make utils_net looks more clear.

Comment (balamuruhans) [inline virttest/utils_net.py]: @xutian sure. I will write another function `get_remote_host_net_ifs()`

**Comment by balamuruhans** [comment]
@xutian, have written another function `get_remote_host_net_ifs()` as suggested.

**Comment by xutian** [comment]
LGTM

**Comment by xutian** [comment]
works for me, merged thanks @balamuruhans

**Pull request merged**